docs/src/technologies/x86_pcie_crosvm.md

```diff
@@ -0,0 +1,44 @@
+<!--
+    Copyright 2022-2023 TII (SSRC) and the Ghaf contributors
+    SPDX-License-Identifier: CC-BY-SA-4.0
+-->
+
+# X86 PCIe device passthrough with crosvm
+
+
+## Enabling PCIe Devices for VFIO with using driverctl tool
+As with other passthroughs we first need to set the target device to use vfio
+ driver. This can be done manually or by using `driverctl` tool as below. Note
+ that running driverctl requires root permissions.
+```
+export BUS="0000:01:00.0"
+driverctl set-override ${BUS} vfio-pci
+```
+
+Starting crosvm as an example. In some cases crosvm may need some privileged
```

Comment:
Replace with:

Let's consider the example of starting crosvm.

In some cases, crosvm may need privileged permissions to work properly. This applies especially for passthrough hardware devices as VFIO devices are generally owned by the root user. For simplicity, it may be easier to run crosvm as the root user but it should be possible to set up correct permissions so that running as root is not needed.

crosvm expects the device's system path as its `--vfio` argument. The argument can differ depending on how the passthrough device is referenced in QEMU. Using the  `guest-address` option is not strictly required by the source documentation but it gives a bit more control for handling the passthrough device on the guest side.

Reply:
"The argument can differ depending on how the passthrough device is referenced in QEMU." Good rephrasing but also changes the meaning a bit. I rewrote it a bit in edit.